### ISSUE TITLE
HMR not reflecting the changes made in reducers

### ISSUE THREAD

**Issue by sabbiu** [zeit/next.js#11794]
# Bug report

## Describe the bug

Making changes to reducers doesn't get replaced as promised by Hot Module Replacement. Only after refreshing the browser, the changes are seen.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Clone [with-redux-thunk](https://github.com/zeit/next.js/tree/canary/examples/with-redux-thunk)
2. Run `nest` or `npm run dev`
3. Make changes to reducers (while the project is still running)
4. Changes are seen only after the page is refreshed

## Expected behavior

- When any changes are made to reducers, it must be reflected in the browser.
- For the time being, is there a way to cancel HMR, so that the browser refreshes on every change.

## Screenshots

Showing the bug --GIF Demo
![GIF: Demo](https://user-images.githubusercontent.com/19733021/78939140-4c98ec80-7ad3-11ea-9f52-62c57feeb813.gif)

## System information

- OS: Tested only in Linux Mint
- Browser (if applies) Tested only in chrome
- Version of Next.js: 9.3.4

## Additional context

Similar Issue: [https://github.com/zeit/next.js/issues/2902](https://github.com/zeit/next.js/issues/2902)

**Comment by lfades** [comment]
Fixed by https://github.com/zeit/next.js/pull/11816 - If the issue persists, comment it here and I'll re open the issue.

Thank you for the report :relieved:

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
